master/manifests/calico-etcd.yaml

```diff
@@ -4,5 +4,4 @@ layout: null
 {% helm %}
```

Comment:
This diff is confusing - I didn't move these, I just deleted the unused ALP manifests.